### PR TITLE
Fixed tests namespaces

### PR DESCRIPTION
**Pull request by ViniTou**

### CODE REVIEW

**Comment by mnocon** [comment]
It need a companion PR for BehatBundle, it's used in https://github.com/ezsystems/BehatBundle/blob/8.3/behat_ibexa_oss.yaml#L120 (and other behat_ibexa_<edition> files).

Also https://github.com/ezsystems/ezplatform-user/blob/2.3/behat_suites.yml#L9 needs to be adjusted

**Comment by ViniTou** [comment]
@mnocon 
Thanks for info, done in:
https://github.com/ezsystems/BehatBundle/pull/202
and
https://github.com/ezsystems/ezplatform-user/pull/96/commits/2acb8d2554ba6ca1a014e1e7760c23bf54404bff

**Review by mnocon — APPROVED** [review]
+1 Behat-wise

File: tests/bundle/DependencyInjection/Configuration/Parser/ChangePasswordTest.php [hunk 1]
```diff
@@ -6,7 +6,7 @@
  */
 declare(strict_types=1);
 
-namespace Ibexa\Platform\Tests\Bundle\OAuth2Client\DependencyInjection\Configuration\Parser;
+namespace EzSystems\EzPlatformUserBundle\Tests\DependencyInjection\Configuration\Parser;
```
Comment (Steveb-p) [inline tests/bundle/DependencyInjection/Configuration/Parser/ChangePasswordTest.php:9]: Isn't this reverting a namespace change from Ibexa to EzSystems?

If so, why is this?

Comment (ViniTou) [reply tests/bundle/DependencyInjection/Configuration/Parser/ChangePasswordTest.php:9]: Its not reverting, its correcting entirely wrong namespace. 

I could introduce new one here, but I am not sure if this have any value in that case. Those class already exists on 2.3 branch and fixing in that way is less convoluted than introducing new one.

**Review by adamwojs — APPROVED** [review]

**Review by Steveb-p — APPROVED** [review]

**Pull request merged**